packages/cli/src/commands/account/new.ts

```diff
@@ -38,13 +39,23 @@ export default class NewAccount extends LocalCommand {
       description:
         "Language for the mnemonic words. **WARNING**, some hardware wallets don't support other languages",
     }),
+    mnemonic: flags.string({
+      description:
+        'Instead of generating a new mnemonic (seed phrases) the user can set the mnemonic to be used. It is required to set it as a string with all the words in order, separated by spaces (example: "word1 word2 word3 ... word24"). If the words are in other language than enaglish, the --language flag must be used. Should be a bip39 mnemonic',
```

Comment:
```suggestion
        'Instead of generating a new mnemonic (seed phrase), use the user-supplied mnemonic instead. The mnemonic should contain all words separate by a space (example: "word1 word2 word3 ... word24"). If the words are a language other than English, the --language flag must be used. Only bip39 mnemonics are supported',
```